README.md

```diff
@@ -96,6 +97,28 @@ complete (and more colorful) variants, see
 [here](https://github.com/seebi/dircolors-solarized) or
 [here](https://github.com/trapd00r/LS_COLORS).
 
+## Parallel Command Execution
+If the `--exec` flag is specified alongside a command template, a job pool will be created for
+generating and executing commands in parallel with each discovered path as the inputs. The syntax
+for generating commands is similar to that of GNU Parallel:
+
+- **{}**: A placeholder token that will be replaced with the discovered path.
+- **{.}**: Removes the extension from the path.
+- **{/}**: Uses the basename of the discovered path.
+- **{//}**: Uses the parent of the discovered path.
+- **{/.}**: Uses the basename, with the extension removed.
+
+```sh
+# Demonstration of parallel job execution
+fd '*.flac' --exec 'sleep 1; echo $\{SHELL}: {}'
```

Comment:
Minor comment: By default, *fd* treats the pattern as a regular expression, not a glob. So this should be either `fd -e flac --exec ...` or `fd '\.flac$' --exec`. Same for the other examples.